### ISSUE TITLE
Are you training the entire YOLO network or freezing the first few layers and just training the last few layers ?

### ISSUE THREAD

**Issue by andohuman** [experiencor/basic-yolo-keras#196]

**Comment by alexcdot** [comment]
https://github.com/AlexeyAB/darknet#how-to-improve-object-detection Freezing the first few layers. This readme helps!

**Comment by dejongyeong** [comment]
Hi @alexcdot, I am new to YOLO and much appreciated it if you could give me some pointers on how to freeze the first few layers for transfer learning and what to add in the `yolov4-custom.cfg` please. Thanks so much.

**Comment by alexcdot** [comment]
https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects this section of the readme provides a step-by-step

**Comment by dejongyeong** [comment]
@alexcdot, thanks for pointing it out and I am following the step-by-step tutorial. I have one question here that I would like to seek clarification on. After training on the custom dataset: cargo containers, cranes, etc (labels not found in the coco dataset) using the pre-trained weights `yolov4.conv.137`, will the `_best.weights` file be able to detect classes found in the coco dataset as well, i.e., person, cars, etc? Or it will just detect the custom dataset labels only?

**Comment by alexcdot** [comment]
It will only detect the custom dataset labels only. To detect the coco dataset classes, you could either just run a separate, pre-trained YOLO model, or retrain with the COCO dataset
